packages/ui/vanilla-ui/src/CrossmintPaymentElement.ts

```diff
@@ -79,14 +85,21 @@ export class CrossmintPaymentElement extends LitElement {
 
     const onEvent = getOnEventFunction(this.onEvent);
 
-    const { listenToEvents } = crossmintPaymentService({ 
-      clientId: this.clientId || this.collectionId,
+    const { listenToEvents } = crossmintPaymentService({
+      clientId: this.clientId,
+      collectionId: this.collectionId,
```

Comment:
`crossmintPaymentService` handles selecting the correct one, so we dont need to do it here